gwf/backends/__init__.py

```diff
@@ -0,0 +1,12 @@
+from __future__ import absolute_import
+from __future__ import print_function
+
+from gwf.backends.slurm import SlurmBackend
+
+
+# This will be set in the gwf script and refer to the grid backend used.
+BACKEND = None
+
+AVAILABLE_BACKENDS = {
+    'slurm': SlurmBackend,
```

Comment:
Yeah, these new commits are all "old" (from before we agreed on the new architecture). More changes will come today :)